netweaver/setup/install_pydbapi.sls

```diff
@@ -0,0 +1,53 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+
+{% for node in netweaver.nodes if node.host == host and node.sap_instance in ['db', 'pas', 'aas'] %}
+{% if loop.first %}
+{% set pydbapi_output_dir = '/tmp/pydbapi' %}
+nw_install_python_pip:
+  pkg.installed:
+    {% if grains['pythonversion'][0] == 2 %}
+    - name: python-pip
+    {% else %}
+    - name: python3-pip
+    {% endif %}
+    - retry:
+        attempts: 3
+        interval: 15
+    - resolve_capabilities: true
+
+nw_extract_pydbapi_client:
+  hana.pydbapi_extracted:
+    - name: PYDBAPI.TGZ
+    - software_folders: {{ netweaver.additional_dvds }}
+    - output_dir: {{ pydbapi_output_dir }}
+    - hana_version: '20'
```

Comment:
@arbulu89 why do you block explicitly to HANA 2 here? I think for HANA 1 the process was exactly the same...

Reply:
The `20` is used during the automatic finding of the folder. The patter of the HANA client is different for HANA 1 and HANA 2. We need to somehow specify this. Otherwise, but happens, if somehow you provide HANA 1 and HANA 2 software folders? It would just return the first match.
The discussion should be if this could be paremetrized. And yes, it could be. I could just set other pillar variable putting `20` as default

Reply:
I think it is fine for now.